contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -223,13 +223,23 @@ def process_assessment_metadata(ccnode, kolibrinode):
     assessment_items = ccnode.assessment_items.all().order_by('order')
     exercise_data = json.loads(ccnode.extra_fields) if ccnode.extra_fields else {}
 
-    mastery_model = {'type' : exercise_data.get('mastery_model') or exercises.M_OF_N}
+    mastery = exercise_data.get('mastery_model') or exercises.M_OF_N
     randomize = exercise_data.get('randomize') or True
     assessment_item_ids = [a.assessment_id for a in assessment_items]
 
-    if mastery_model['type'] == exercises.M_OF_N:
-        mastery_model.update({'n':exercise_data.get('n') or min(5, assessment_items.count()) or 1})
-        mastery_model.update({'m':exercise_data.get('m') or min(5, assessment_items.count()) or 1})
+    if mastery == exercises.M_OF_N:
```

Comment:
Instead of doing this globally (which it seems also impacts the mastery model exported to the assessmentmetadata), could you make a separate variable called `legacy_mastery_model` which we use to write into the `exercise.json`, but leave the assessmentmetadata exported mastery_model the same?